fastats/core/decorator.py

```diff
@@ -44,9 +40,16 @@ def fs_wrapper(*args, **kwargs):
 
             new_funcs = {}
             for v in kwargs.values():
-                if isfunction(v) and v.__name__ not in kwargs:
+                if v.__name__ in kwargs:
+                    continue
+
+                if isfunction(v):
                     new_funcs[v.__name__] = convert_to_jit(v)
-            kwargs = {k: convert_to_jit(v) for k, v in kwargs.items()}
+                elif isbuiltin(v):
+                    new_funcs[v.__name__] = v
```

Comment:
`math` functions are not regarded as functions by `isfunction`.

Reply:
Python 3.6.3 on Linux

```python
>>> import math
>>> import inspect
>>> inspect.isfunction(math.tanh)
False
>>> inspect.isbuiltin(math.tanh)
True
```

Reply:
Yea, nice catch, thanks!